astroplan/data/observatories.json

```diff
@@ -12,9 +11,8 @@
     }, 
     "NOV": {
         "aliases": [
-            "National Observatory of Venezuela", 
-            "NOV"
```

Comment:
Did you remove those on purpose?
(I don't care either way ... just making sure before I merge this.)